README.md.pre-ubuntu18

```diff
@@ -0,0 +1,338 @@
+# ENCODE Metadata Database
```

Comment:
Are we still supporting ENCODE/SNOVAULT pre-ubuntu18? If no, do we need this page?

Reply:
I was going to keep it for a bit.  There are few external users of the repo who may still be on ubuntu14.  Although we could very well be the last people to use it!